packages/api-sdk/src/medical/models/demographics.ts

```diff
@@ -46,19 +47,20 @@ export type PersonalIdentifier = z.infer<typeof personalIdentifierSchema>;
 
 export const genderAtBirthSchema = z.enum(["F", "M"]);
 
-const phoneLength = 10;
 export const contactSchema = z
   .object({
     phone: z.coerce
       .string()
-      .transform(phone => stripNonNumericChars(phone))
-      .refine(phone => phone.length === phoneLength, {
-        message: `Phone must be a string consisting of ${phoneLength} numbers. For example: 4153245540`,
+      .transform(phone => normalizePhoneNumber(phone))
+      .refine(isPhoneValid, {
+        message: `Phone must be a string consisting of ${phoneLength} numbers. For example: ${examplePhoneNumber}`,
       })
-      .or(defaultOptionalString),
-    email: z.string().email().or(defaultOptionalString),
+      .or(z.null())
+      .or(z.undefined()),
+    email: z.string().email().or(z.null()).or(z.undefined()),
```

Comment:
From my perspective, in the HTTP layer `null` and "not present" are the same thing (because we're not using `PATCH`), they represent the lack of value for that property. Surely, `null` needs to be converted to `undefined`, as we don't use `null` inside the app.

We could argue that we don't want to accept `null` on the HTTP/route layer so we can use PATCH in the future, but:
- we already use `.nullish()` across the board (that's null + undefined), I just had to set them separately b/c the're inside `.or()`;
- if we start to work w/ PATCH, we probably will need to move from v1 to v2 of the API.

Thoughts?

Reply:
If do it everywhere else, then it's fine -- not ideal but better to be consistent and wrong than inconsistent :).